lib/components/form/connect-location-field.js

```diff
@@ -77,6 +83,7 @@ export default function connectLocationField(
     const stateToProps = {
       currentPosition,
       geocoderConfig,
+      isValid,
```

Comment:
Won't hold this agains this PR, but the invalid state of the underlying `<input>` is not "seen" in ARIA dev tools.

Reply:
This might actually be an issue with ARIA dev tools that does not render the invalid flags on non-standard elements.